web/src/views/DandisetLandingView/ShareDialog.vue

```diff
@@ -104,11 +118,12 @@ export default defineComponent({
       }
       return '';
     });
+    const doiLink = computed(() => (meta.value?.doi ? `https://doi.org/:${meta.value?.doi}` : ''));
```

Comment:
extra `:` snuck in. otherwise the output looks great.

```suggestion
    const doiLink = computed(() => (meta.value?.doi ? `https://doi.org/${meta.value?.doi}` : ''));
```

Reply:
fixed in fecc64fa5ddaf237d51a16a46dd8f8d66e18c37d